docs/02.API-REFERENCE.md

```diff
@@ -3600,7 +3648,7 @@ main (void)
 
 # Exception manipulation functions
 
-*Changed in version 2.0*: The exception handling and manipulation was modified and the old methods were replaced.
+*Updated in version 2.0*: The exception handling and manipulation was modified and the old methods were replaced.
```

Comment:
reckless nitpicking :)  "The exception handling and manipulation have been modified, and the old methods were replaced.